portfolio/src/main/webapp/index.html

```diff
@@ -8,12 +8,42 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
+      <h1>Rishith Kyatham Personal Portfolio</h1>
+      <p>I am a Sophomore in Computer Science double Majoring in 
+        Computer Science and Applied Mathematics & Science. 
+        My interest leans  Machine Learning and I am aiming to become an aspiring Full-Stack Developer in the future.</p>
+      <p>I am Currently in search for internships for this Summer and next Summer. 
+        My primary strengths exist in Backend Web Development.
+        I am looking more into Cloud Development currently.</p>
+      <p>Click for Greeting!</p>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
     </div>
-  </body>
+    <a href="images/pic.jpg"><img src="images/pic.jpg"/></a>
+
+    <h2>Some Projects I Worked On</h2>
+    <p>GEO Data Visualizer</p>
+    <p>Developed a Web Application that prompts user for a location, 
+    contacts an external API to fetch latitude and 
+    longitude data, and visualizes it in Google Maps using Javascript.
```

Comment:
nitnitnit: also indent here like you did for lines 13-14.

Reply:
Got it

Reply:
Finished the edits.